examples/postgres/locustfile.py

```diff
@@ -0,0 +1,36 @@
+from locust import TaskSet, between, task
+from locust.contrib.postgres import PostgresUser
+
+import os
+import random
+
+
+class UserTasks(TaskSet):
+    @task
+    def run_select_query(self):
+        self.client.execute_query(
+            "SELECT * FROM loadtesting.invoice WHERE amount > 500",
+        )
+
+    @task(3)
+    def run_update_query(self):
+        random_amount = random.randint(1, 12)
+        self.client.execute_query(
+            f"UPDATE loadtesting.invoice SET amount={random_amount} WHERE amount < 10",
+        )
+
+
+class PostgresLocust(PostgresUser):
+    tasks = [UserTasks]
+    min_wait = 0
```

Comment:
remove the wait_time stuff and just leave it as the default, to simplify the example.

You can skip the taskset and put the tasks directly under the user too.